lib/apiservers/engine/backends/container_proxy.go

```diff
@@ -709,6 +709,9 @@ func (c *ContainerProxy) AttachStreams(ctx context.Context, vc *viccontainer.Vic
 func dockerContainerCreateParamsToPortlayer(cc types.ContainerCreateConfig, layerID string, imageStore string) *containers.CreateParams {
 	config := &models.ContainerCreateConfig{}
 
+	config.NumCpus = &cc.HostConfig.CPUCount
+	config.Memory = &cc.HostConfig.Memory
```

Comment:
Docker does everything in KiB/MiB/GiB but I guess it doesn't matter since (apparently) VC only cares if the memory you allocate to a VM is a multiple of 4. 👍